sklearn/model_selection/_split.py

```diff
@@ -1865,6 +1866,80 @@ def get_n_splits(self, X=None, y=None, groups=None):
         return len(self.unique_folds)
 
 
+class BinnedStratifiedKFold(StratifiedKFold):
+    """Stratified K-Folds cross-validator
+
+    Provides train/test indices to split data in train/test sets.
+
+    This cross-validation object is a variation of KFold that returns
+    stratified folds by binning a regression target. The folds are made
+    by approximately preserving the distribution of targets in each bin.
+
+    Read more in the :ref:`User Guide <cross_validation>`.
```

Comment:
Shouldn't this link to the Binned Stratified k-fold section of the user guide?

Reply:
all splitters refer to the main page right now? We might split this up but then should probably update the links everywhere?